tests/pulsar-kafka-compat-client-test/src/test/java/org/apache/pulsar/tests/integration/compat/kafka/PulsarKafkaProducerThreadSafeTest.java

```diff
@@ -37,8 +36,8 @@ private String getPlainTextServiceUrl() {
         return container.getPlainTextServiceUrl();
     }
 
-    @Override
-    public void setUpCluster() throws Exception {
+    @BeforeTest
+    private void setupThreadSafeTest() {
```

Comment:
This change is unrelated can you please revert?

Cc @dlg99 @lhotari

Reply:
Removed this from here.